scripts/test.sh

```diff
@@ -6,9 +6,14 @@ if [ -z "$TEST_GREP" ]; then
 fi
 
 node="node"
+runSequentially=""
 
 if [ "$TEST_DEBUG" ]; then
    node="node --inspect --debug-brk"
 fi
 
-$node node_modules/mocha/bin/_mocha `scripts/_get-test-directories.sh` --opts test/mocha.opts --grep "$TEST_GREP"
+if [ "$CI" ]; then
```

Comment:
Wait what, this works!?

I thought you needed to give `-n` to make it check for string emptiness. Just to be sure and ensure this won't break if the string starts with a `-`, it's best to write `[ -n "$CI" ]`.